VRDR.CLI/Program.cs

```diff
@@ -910,5 +986,132 @@ private static string Truncate(string value, int length)
                 return value.Substring(0, length);
             }
         }
+ private static IJEMortality mre2ije(string mrefilename){

```

Comment:
Can you get back in the habit of code formatting to use our coding style for new code (brackets are always on a line on their own)?